src/components/CustomForm.js

```diff
@@ -53,7 +53,7 @@ const CustomForm = ({ status, message, onValidated }) => {
               placeholder="Name"
               required
             />
-            <label for="Name" className={styles.formLabel}>
+            <label htmlFor="Name" className={styles.formLabel}>
```

Comment:
Nice catch 😁

Reply:
Thank the Chrome debug console, not me haha